src/Features/CSharp/Portable/MakeMethodAsynchronous/CSharpMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -81,7 +81,7 @@ protected override bool IsAsyncSupportingFunctionSyntax(SyntaxNode node)
             {
                 if (!keepVoid)
                 {
-                    newReturnType = knownTypes.TaskType.GenerateTypeSyntax();
+                    newReturnType = knownTypes._taskType.GenerateTypeSyntax();
```

Comment:
note, if these are public readonlys, they don't need to be named `_...`

Reply:
That seems to be the configured naming convention

Reply:
sounds liek a mistake.  IDE has lots of of public readonly fields that are not underscored.  underscores are intended to only be for private members.  @sharwell do you know what's up here?